### PR TITLE
bump/helmt-5.0.2-add-unit-test

### PR DESCRIPTION
**Pull request by hilleer**

### CODE REVIEW

**Comment by hilleer** [comment]
@ebdrup feel free to review 😄

File: test/index.spec.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const httpsonly = require('../lib');
+const mocks = require('node-mocks-http');
+
+describe('index', () => {
+	describe('when middleware is added', () => {
+		it('should call next function', () => {
+			const { req, res } = mocks.createMocks();
+
+			let actuallyCalledNext = false;
+			const next = () => actuallyCalledNext = true;
```
Comment (soulmonk) [inline test/index.spec.js:10]: why not sinon.stub? )

**Review by soulmonk — APPROVED** [review]
LGTM

File: test/index.spec.js (hunk 1 above)
Comment (hilleer) [reply test/index.spec.js:10]: this is simpler than sinon IMO

File: lib/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+module.exports = () => httpsOnlyMiddleware;
+
+function httpsOnlyMiddleware(req, res, next) {
+	if (req.headers['x-forwarded-proto'] === 'http') {
+		res.redirect(301, 'https://' + req.headers.host + req.url);
+		return;
+	}
+	helmet.hsts({
+		maxAge: ONE_YEAR,
+		includeSubDomains: true,
```
Comment (hilleer) [inline lib/index.js:14]: Fixed a typo/breaking change [here](https://github.com/debitoor/httpsonly/pull/1/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346L11) as it was previously spelled like `includeSubdomains`.

Comment (soulmonk) [reply lib/index.js:14]: I think this is not "breaking change"
before: https://github.com/helmetjs/hsts/blob/v1.0.0/index.js#L37-L39
after: https://github.com/helmetjs/helmet/blob/main/middlewares/strict-transport-security/index.ts#L31-L35

**Pull request merged**